tensorflow/tools/ci_build/Dockerfile.cpu

```diff
@@ -13,3 +13,11 @@ RUN /install/install_bazel.sh
 # Set up bazelrc.
 COPY install/.bazelrc /root/.bazelrc
 ENV BAZELRC /root/.bazelrc
+
+# Script for (optional) test on install
+COPY builds/test_on_install.sh /builds/test_on_install.sh
+RUN chmod a+rx /builds/test_on_install.sh
+
+# Grant users installation permission on Python package directory
+RUN chmod -R a+rw /usr/local/lib/python2.7/dist-packages
```

Comment:
Can it be installed into user home? It would make testing easier with permissions for people even outside docker. And we should try hard to make it simple and the same inside docker as well as outside.